api/src/pcapi/scripts/algolia_settings.py

```diff
@@ -0,0 +1,119 @@
+import json
+import logging
+import os
+
+from algoliasearch.search_client import SearchClient
+from algoliasearch.search_index import SearchIndex
+import click
+
+from pcapi import settings as pcapi_settings
+from pcapi.utils.blueprint import Blueprint
+
+
+ALGOLIA_SETTINGS_DIR = os.path.abspath(
+    os.path.normpath(
+        os.path.join(
+            os.path.dirname(__file__),
+            os.path.pardir,
+        )
+    )
+)  # pass-culture-main/api/src/pcapi
```

Comment:
(détail) Variante, plus simple (et qui ne dépend pas de l'emplacement du fichier depuis lequel on l'exécute) : 

```suggestion
import pcapi

ALGOLIA_SETTINGS_DIR = pathlib.Path(pcapi.__path__[0])
```

Reply:
c'est bien plus clair comme, ça me plait beaucoup